core/ipc/rpc/kafka/src/main/java/org/opennms/core/ipc/rpc/kafka/KafkaRpcServerManager.java

```diff
@@ -323,6 +354,9 @@ public void run() {
 
     }
 
+    private Tracer getTracer() {
+        return tracerRegistry.getTracer(minionIdentity.getId()+"@"+minionIdentity.getLocation());
```

Comment:
Can we flip the location and id here so that the all of the systems at the same location a "grouped" together in the Jaeger UI?